mlrun/funcdoc.py

```diff
@@ -196,6 +196,11 @@ def ann_type(ann):
         name = ann.value.id
         inner = ', '.join(ann_type(e) for e in iter_elems(ann.slice))
         return f'{name}[{inner}]'
+    
+    if isinstance(ann, ast.Attribute):
+        name = ann.attr
+        mod = ann.value.id
+        return f'{mod}.{name}'
```

Comment:
@tebeka we only want to store the name (i.e.  `DataItem`) without the module, also did u test the use case with Union etc.?

Reply:
@yaronha Currently in the case of union it returns `Union[DataItem, int]`, is that OK?